packages/suite/src/storage/migrations/index.ts

```diff
@@ -781,4 +783,96 @@ export const migrate: OnUpgradeFunc<SuiteDBSchema> = async (
         // @ts-expect-error
         backendSettings.delete('trop');
     }
+
+    if (oldVersion < 37) {
+        const accountsStore = transaction.objectStore('accounts');
+        await accountsStore
+            .openCursor()
+            .then(async function updateMetadataKeys(cursor): Promise<void> {
+                if (!cursor) {
+                    return;
+                }
+                const account = cursor.value;
+
+                // @ts-expect-error
+                if (!account.metadata?.fileName) {
+                    return; // is this ok? return here?
+                }
+                account.metadata = {
+                    key: account.metadata.key,
+                    1: {
+                        // @ts-expect-error
+                        fileName: account.metadata.fileName,
+                        // @ts-expect-error
+                        aesKey: account.metadata.aesKey,
+                    },
+                };
+                await cursor.update(account);
+
+                return cursor.continue().then(updateMetadataKeys);
+            });
+
+        const devicesStore = transaction.objectStore('devices');
+        await devicesStore
+            .openCursor()
+            .then(async function updateMetadataKeys(cursor): Promise<void> {
+                if (!cursor) {
+                    return;
+                }
+                const device = cursor.value;
+
+                if (
+                    device.metadata.status === 'enabled' &&
+                    // @ts-expect-error
+                    device.metadata.fileName &&
+                    // @ts-expect-error
+                    device.metadata.aesKey
+                ) {
+                    device.metadata[1] = {
+                        // @ts-expect-error
+                        fileName: device.metadata.fileName,
+                        // @ts-expect-error
+                        aesKey: device.metadata.aesKey,
+                    };
+                }
+                device.metadata = {
+                    status: 'disabled',
+                };
```

Comment:
This will overwrite any changes from above so all devices' metadata will be `{ status: 'disabled' }` after migration.